[2022-02-06 11:47:12,738] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.col_process_image scheduled__2022-02-05T00:00:00+00:00 [queued]>
[2022-02-06 11:47:12,996] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.col_process_image scheduled__2022-02-05T00:00:00+00:00 [queued]>
[2022-02-06 11:47:13,094] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2022-02-06 11:47:13,133] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2022-02-06 11:47:13,165] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-02-06 11:47:13,536] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): col_process_image> on 2022-02-05 00:00:00+00:00
[2022-02-06 11:47:13,677] {standard_task_runner.py:52} INFO - Started process 200 to run task
[2022-02-06 11:47:13,701] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cpp_workflow', 'col_process_image', 'scheduled__2022-02-05T00:00:00+00:00', '--job-id', '3185', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp99rmi2kp', '--error-file', '/tmp/tmpigb5if_p']
[2022-02-06 11:47:13,840] {standard_task_runner.py:77} INFO - Job 3185: Subtask col_process_image
[2022-02-06 11:47:14,284] {logging_mixin.py:109} INFO - Running <TaskInstance: cpp_workflow.col_process_image scheduled__2022-02-05T00:00:00+00:00 [running]> on host 218df5ddc363
[2022-02-06 11:47:15,362] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cpp_workflow
AIRFLOW_CTX_TASK_ID=col_process_image
AIRFLOW_CTX_EXECUTION_DATE=2022-02-05T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-05T00:00:00+00:00
[2022-02-06 11:47:15,699] {python.py:152} INFO - Done. Returned value was: None
[2022-02-06 11:47:15,815] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=cpp_workflow, task_id=col_process_image, execution_date=20220205T000000, start_date=20220206T114712, end_date=20220206T114715
[2022-02-06 11:47:16,088] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-06 11:47:17,115] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
